content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Cloudify cluster day 2 operations
+description: Guides for manging Cloudify HA clusters
+weight: 80
+alwaysopen: false
+---
+
+## Introduction
+
+Cloudify HA solution can contains three different clusters of database nodes, message queue brokers and Cloudify Managers.
+This require the ability to manage those cluster, which include add/removing/listing/custom-action to the cluster.  
+
+## Cloudify management service operations  
+
+* For removing a management service cluster node, please run on a management node (not the one you wish to remove):
+
+```bash  
+   cfy cluster remove <A resolvable hostname>
+``` 
+
+* For adding a management service cluster node, please install the new node on the same network settings as in Cloudify management service nodes.
+
+## Message queue cluster operations
+
+* A new broker node (an installed node) can be added to the cluster by running the following:
+
+  * Run the following command to connect the broker to the cluster:
+  
+    ```bash  
+       cfy_manager brokers add -j\--join-node <A resolvable hostname which complies to the format rabbit@<hostname> >
```

Comment:
I'm fairly sure you don't need to provide the rabbit@ prefix, that'll be added automatically if there's no @ in the hostname.

Reply:
However, it does need to target an existing member of the cluster (if memory serves- the help doc should confirm)